lib/bookingsync/api/client.rb

```diff
@@ -209,7 +214,7 @@ def paginate(path, options = {}, &block)
               first_request = false
             end
             break unless response.relations[:next]
-            response = response.relations[:next].get
+            response = response.relations[:next].public_send(request_method)
```

Comment:
Refactored a bit handling these requests for relations, looks better now I guess.